operations/admin_from_mdb.go

```diff
@@ -270,7 +271,6 @@ func fromMdbForLocal() cli.Command {
 					return errors.Wrap(err, "writing buffer to tarball")
 				}
 
-				grip.Error(cursor.Close(ctx))
```

Comment:
Should this be moved to line 275? 
the lint task says: "operations/admin_from_mdb.go:244:5: for loop containing defer will not run until end of function"